### DISCUSSION TITLE
Display Module name in the page title when configuring it in Module Manager

### DISCUSSION THREAD

**Discussion by rateforx** [PrestaShop/PrestaShop#40194] [Ideas]
When you're configuring a module the page title says `Module Manager - My Shop`. When I am configuring multiple modules at the same time on different tabs its impossible to say which tab belongs to which module without checking the tab.

It would be really helpful if the page title was more descriptive eg.: `Module name configuration - Module Manager - My Shop` or just `Module name configuration - My Shop`.

I've tried fixing the issue myself by adding a "title" block to the admin `header.tpl` template and then prepending the module name to the title but it seems that the `configure.tpl` template does not extend the header properly.

**Comment by mariem-abid** [comment]
Hi @rateforx,

What is the exact version PHP do you use?

Thanks!

**Comment by rateforx** [comment]
Hi, sorry for the delay. I am using 7.2.18. What does it have to do with the case?

**Comment by khouloudbelguith** [comment]
Hi @rateforx,

Thanks for your report.
It could be a good idea to have a page title specified for each module & not a general title -` Module Manager - Shop Name`
![image](https://user-images.githubusercontent.com/16067358/59515733-b2b6e200-8eb7-11e9-92e4-97704a6c840a.png)
![image](https://user-images.githubusercontent.com/16067358/59515919-1d681d80-8eb8-11e9-9673-3ed48db5c169.png)

@marionf, @colinegin what do you think?

Thanks!

**Comment by NeOMakinG** [comment]
Looks like the meta_title should be defined by the backend module configuration controller, can't do anything on the front-end part
